### PR TITLE
Desktop: Development: Clarifying that dev mode is started from CLI

### PR DESCRIPTION
**Pull request by amandamcg**

### CODE REVIEW

**Review by tessus — COMMENTED** [review]
LGTM

File: readme/api/references/development_mode.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Development mode
 
-When experimenting with Joplin, for example when developing a plugin or trying a theme, you might want to run Joplin in development mode. Doing so means that Joplin will run using a different profile, so you can experiment without risking to accidentally change or delete your data.
+When experimenting with Joplin, for example when [developing a plugin](https://joplinapp.org/api/get_started/plugins/) or trying a theme, you might want to run Joplin in development mode. Doing this will open a second copy of Joplin using a different profile with test notes and notebooks, so you can experiment without risking changing or deleting your data.
 
-To enable Development Mode, open Joplin as normal, then go to **Help => Copy dev mode command to clipboard**. This will copy a command to the clipboard. Now close Joplin, and start it again in dev mode using the command you've just copied.
\ No newline at end of file
+To enable Development Mode, go to `Help => Copy dev mode command to clipboard`. This will copy a command to your clipboard. Paste this command in a shell / terminal to run a development version of the app.
```
Comment (laurent22) [inline readme/api/references/development_mode.md:5]: Any reason why the command was moved from bold formatting to code formatting? You might be right about this actually but I don't know. How is it in other documentations for things like menu item paths? If code block is what's usually used then why not.

**Comment by laurent22** [comment]
@amandamcg, for this pull request too please let me know if you're still interested in completing it.

**Comment by amandamcg** [comment]
Hey yes, sorry for the delay. Will fix and submit.

**Comment by amandamcg** [comment]
Ok, I think I resubmitted. Let me know if that's the right way, I am not super familiar with how Github handles pull requests.

File: readme/api/references/development_mode.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Development mode
 
-When experimenting with Joplin, for example when developing a plugin or trying a theme, you might want to run Joplin in development mode. Doing so means that Joplin will run using a different profile, so you can experiment without risking to accidentally change or delete your data.
+When experimenting with Joplin, for example when [developing a plugin](https://joplinapp.org/api/get_started/plugins/) or trying a theme, you might want to run Joplin in development mode. Doing this will open a second copy of Joplin using a different profile with test notes and notebooks, so you can experiment without risking changing or deleting your data.
 
-To enable Development Mode, open Joplin as normal, then go to **Help => Copy dev mode command to clipboard**. This will copy a command to the clipboard. Now close Joplin, and start it again in dev mode using the command you've just copied.
\ No newline at end of file
+To enable Development Mode, go to **Help => Copy dev mode command to clipboard**. This will copy a command to your clipboard. Paste this command in a shell / terminal to run a development version of the app.
```
Comment (tessus) [inline readme/api/references/development_mode.md:5]: In other files we use `>` as a menu separation character. I know this was already like this before, but I think we should change `=>` to `>` to make it consistent.

**Pull request merged**